components/tensorboard-controller/OWNERS

```diff
@@ -0,0 +1,10 @@
+approvers:
+  - elikatsis
+  - kandrio98
+  - kimwnasptd
+  - quanjielin
+reviewers:
```

Comment:
approvers shouldn't be listed as approvers

Reply:
I think I'm missing something here, where should approvers be listed?

Reply:
My bad typo; meant to say that approvers should be listed as reviewers.

Reply:
I see. Which people should I assign as reviewers?

Reply:
@kimwnasptd You can just leave it blank if there is no one that makes sense to add as a reviewer.